knowledge-base/how-to-load-telerik-report-viewer-resources-locally-in-angular-application-and-use-them-to-customize-the-viewer.md

```diff
@@ -0,0 +1,109 @@
+---
+title: How to load Telerik Report viewer resources locally in Angular application and use them to customize the viewer
+description: In some cases a modification of the default behavior, layout or styles of the HTML5 Report Viewer in an Angualr application is needed. This article elaborates further on how to load the required default resources locally.
+type: how-to
+page_title: How to load Telerik Report viewer resources locally in Angular application and use them to customize the viewer
+slug: how-to-load-telerik-report-viewer-resources-locally-in-angular-application-and-use-them-to-customize-the-viewer
+position: 
+tags: AngularReportViewer
+ticketid:
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>13.0.19.222</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+	    <tr>
+	    	<td>Project Type</td>
+	    	<td>Angular application</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+Do you want to customize the report viewer's default behavior? Let's find out how.
```

Comment:
Maybe "layout and behavior" or similar to involve also the pure appearance of the widget